kato/kato-azure/src/main/groovy/com/netflix/spinnaker/kato/azure/deploy/AzureOperationPoller.groovy

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.kato.azure.deploy
+
+import com.netflix.spinnaker.kato.data.task.Task
+import org.springframework.beans.factory.annotation.Autowired
+
+class AzureOperationPoller {
+
```

Comment:
That code is not actually used at the moment. We tried to follow the Google's overall implementation and they had a comment in theirs that this can be used to facilitate testing.

I'm going to remove it though as it is dead code and I don't think we have any plans to use it any time soon.